draft-barnes-mimi-arch.md

```diff
@@ -298,6 +305,34 @@ Client        Follower        Hub         Follower        Client
 ~~~~~
 {: #fig-protocols title="MIMI Protocols" }
 
+## End-to-End Security
+
+As noted above, all of the clients participating in a room are part of the same
+end-to-end security context.  This allows them to encrypt and integrity-protect
+their messages so that they are secure from inspection or tampering as they
```

Comment:
secure from inspection for app messages but not Commits/Proposals...

```suggestion
their messages so that they are secure from inspection (of content) or tampering as they
```

Reply:
I think "messages" as in "actual messages sent in the room" cabins this appropriately.